tools/static-assets/skel-full/imports/api/links/methods.js

```diff
@@ -1,19 +1,24 @@
 // Methods related to links
 
 import { Meteor } from 'meteor/meteor';
-import { Links } from './links.js';
 import { check } from 'meteor/check';
+import { Links } from './links.js';
 
 Meteor.methods({
   'links.insert'(title, url) {
     check(url, String);
     check(title, String);
 
     // Check if this is a valid url
-    if (!url.match(/((http|https)\:\/\/)+[a-zA-Z0-9\.\/\?\:@\-_=#]+\.([a-zA-Z0-9\&\.\/\?\:@\-_=#])*/g)){
-      throw new Meteor.Error('Bad url. I.e https://www.meteor.com');
+    const re = /((http|https)\:\/\/)+[a-zA-Z0-9\.\/\?\:@\-_=#]+\.([a-zA-Z0-9\&\.\/\?\:@\-_=#])*/g;
+    if (!url.match(re)) {
```

Comment:
[`RegExp.prototype.test()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/test) has long been proven to be more performant than [`String.prototype.match()`](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/String/match) in the browser.  These days, I find them to be the same in the Node environment, but for the sake of consistency I'd recommend changing this to:

`if (!re.test(url)) {`